internal/cmds/iter_test.go

```diff
@@ -0,0 +1,13 @@
+//go:build go1.23
+
+package cmds
+
+import (
+	"maps"
+)
+
+func iter0(s Builder) {
+	s.Hmset().Key("1").FieldValue().FieldValues(maps.All(map[string]string{"1": "1"})).Build()
+	s.Hset().Key("1").FieldValue().FieldValues(maps.All(map[string]string{"1": "1"})).Build()
+	s.Xadd().Key("1").Id("*").FieldValue().FieldValues(maps.All(map[string]string{"1": "1"})).Build()
```

Comment:
We need to call this function somewhere in tests.

Reply:
thanks, I fixed it.